examples-extra/basic/sdk/src/generatedNoCheck/ontology/objects/BoundariesUsState.ts

```diff
@@ -11,9 +11,22 @@ export interface BoundariesUsState
   primaryKeyApiName: 'usState';
   primaryKeyType: 'string';
   properties: {
+    /**
+     *   display name: 'Geometry10M',
+     *   description: geoshape
+     */
     geometry10M: PropertyDef<'geoshape', 'nullable', 'single'>;
+    /**
+     * (no ontology metadata)
```

Comment:
Do you feel like it'd be better to just not have anything here? Not sure how much no ontology metadata adds besides cluttering the generated files

Reply:
I went with this so there wouldn't be confusion about there possibly being a bug that there was no ontology data. Happy for us to change this in a follow up.